cmd/Attribution.txt

```diff
@@ -0,0 +1,28 @@
+The following open source projects are referenced by Modbus Device Service Go:
+
+goburrow/modbus (unspecified) - https://github.com/goburrow/modbus
+https://github.com/goburrow/modbus/blob/master/LICENSE
+
+tbrandon/mbserver (MIT) - https://github.com/tbrandon/mbserver
+https://github.com/tbrandon/mbserver/blob/master/LICENSE
+
+BurntSushi/toml (MIT) - https://github.com/BurntSushi/toml
+https://github.com/BurntSushi/toml/blob/master/COPYING
+
+gorilla/mux 1.6.2 (BSD-3) - https://github.com/gorilla/mux
+https://github.com/gorilla/mux/blob/master/LICENSEE
+
+hashicorp/consul 1.1.0 (Modzilla Public License 2.0) - https://github.com/hashicorp/consul
+https://github.com/hashicorp/consul/blob/master/LICENSE
+
+robfig/cron (MIT) - https://github.com/robfig/cron
+https://github.com/robfig/cron/blob/master/LICENSE
+
+mgo v2 (BSD-2) - https://github.com/go-mgo/mgo
+https://github.com/go-mgo/mgo/blob/v2-unstable/LICENSE
+
+mgo/bson v2 (unspecified) - https://github.com/go-mgo/mgo/tree/v2/bson
+https://github.com/go-mgo/mgo/blob/v2/bson/LICENSE
+
+gopkg.in/yaml v2 (Apache 2.0) - https://github.com/go-yaml/yaml/tree/v2
+https://github.com/go-yaml/yaml/blob/v2/LICENSE
```

Comment:
This file also needs to include a direct reference for `edgex-go` and `device-sdk-go` because it imports those packages.